connectors/spark/src/main/scala/io/unitycatalog/connectors/spark/UCSingleCatalog.scala

```diff
@@ -92,7 +92,13 @@ private class UCProxy extends TableCatalog {
     this.name
   }
 
-  override def listTables(namespace: Array[String]): Array[Identifier] = ???
+  override def listTables(namespace: Array[String]): Array[Identifier] = {
+    if (namespace.length > 1) {
+      throw new ApiException("Unity Catalog does not support nested namespace.")
```

Comment:
1. "nested namespaces" (plural)
2. Can you add what the `namespace` is in the exception

```
Nested namespaces are not supported: [namespace]
```

Reply:
done.